modules/vcf2maf/1.1/vcf2maf.smk

```diff
@@ -95,53 +95,58 @@ rule _vcf2maf_run:
 
 def get_chain(wildcards):
     if "38" in str({wildcards.genome_build}):
-        return reference_files("genomes/{genome_build}/chains/hg38ToHg19.over.chain")
+        return reference_files("genomes/{genome_build}/chains/grch38/hg38ToHg19.over.chain")
     else:
-        return reference_files("genomes/{genome_build}/chains/hg19ToHg38.over.chain")
+        return reference_files("genomes/{genome_build}/chains/grch37/hg19ToHg38.over.chain")
 
 rule _vcf2maf_crossmap:
     input:
         maf = rules._vcf2maf_run.output.maf,
         convert_coord = CFG["inputs"]["convert_coord"],
         chains = get_chain
     output:
-        maf =  CFG["dirs"]["crossmap"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{base_name}.converted.maf"
+        maf =  CFG["dirs"]["crossmap"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{base_name}.converted"
```

Comment:
This is just an empty target file, right? Could you please re-name it e.g. `converted = CFG...`? I think it would help avoid confusion on debugging.

Reply:
That empty file was my suggestion as a sentinel to get around the unpredictable naming of the real output. There's a long Slack thread on this from last night if you want to scroll through the last bit of it for context.

Reply:
The pipeline makes perfect sense, I'm just suggesting it shouldn't be called `maf = ...` since it's not the actual maf file.

Reply:
Yes, this is an empty file. I rename it to `dispatch` so that it will be more indicative of what the file really is.

Reply:
Oh, I see. You just meant the variable name. Sorry.

Reply:
I renamed it to be `dispatch` and it is ready to be merged if approved.